Docs/index.md

```diff
@@ -0,0 +1,46 @@
+# Welcome to the Santa Docs
+
+Santa is a binary whitelisting/blacklisting system for macOS. Here you will find the documentation for understanding how Santa works, how to deploy it and how to contribute.
+
+#### Introduction
+
+The following documents give an overview on how Santa accomplishes binary whitelisting/blacklisting at the enterprise scale.
+
+- [Binary Whitelisting](introduction/binary-whitelisting-overview.md): How Santa makes allow or deny decisions for any `exec()` taking place.
+- [Syncing](introduction/syncing-overview.md): How configuration and whitelist / blacklist rules are applied from a sync-server.
+
+#### Deployment
+
+* [Configuration](deployment/configuration.md): The local and sync-server configuration options.
+
+#### Development
+
+* [Building Santa](development/building.md): How to build and load test builds of Santa on a development machine.
+* [Contributing](../CONTRIBUTING.md): How to contribute a bug fix or new feature to Santa.
+
+#### Details
+
+For those who want even more details on how Santa works under the hood, this section is for you.
+
+######Binaries
+
+There are a five main components that make up Santa. There are documents explaining each piece in detail. Here is a quick one-liner on each component. These quick descriptions do not encompass all the jobs performed by each component, but do provide a quick look at the basic functionality utilized to achieve the goal of binary whitelisting/blacklisting.
```

Comment:
There are five main components that make up Santa whose core functionality is described in snippets below. For additional detail on each component, visit their respective pages.